packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -307,6 +321,7 @@ const getStyles = (theme: GrafanaTheme2) => {
     }),
     titleItems: css({
       display: 'flex',
+      height: '100%',
```

Comment:
I removed the other styling here, probably leftover from the merge. Because it helps with accessibility: https://github.com/grafana/grafana/pull/62466#issuecomment-1408752093